spark/hbase-spark-it/pom.xml

```diff
@@ -282,6 +282,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+    </dependency>
+
```

Comment:
This dependency is used only in the unit tests and only at one place. Is it really required just for generating a random string?

Reply:
Good point. I just moved from the deprecated one to the new one. Completely removed this dependency with the latest changes.